package-lock.json

```diff
@@ -268,6 +268,16 @@
 				"interpolate-components": "^1.1.1",
 				"prop-types": "^15.7.2",
 				"react-stripe-elements": "4.0.2"
+			},
+			"dependencies": {
+				"react-stripe-elements": {
```

Comment:
This sounds unrelated to this change.

I've seen it in other renovate PRs. My theory is that someone forgot to commit a change to package-lock.json and now is popping up in all PRs.

Reply:
Nevermind, already fixed in master in https://github.com/Automattic/wp-calypso/commit/ac131bdbacd2155b8d5ed708e559da557e2f3f61#diff-32607347f8126e6534ebc7ebaec4853d. Probably a rebase will make many changes in package-lock.json go away